cmd/root.go

```diff
@@ -38,14 +38,16 @@ import (
 var json bool
 
 var save bool
-var types diffTypes
+var types multiValueFlag
 var noCache bool
 
 var outputFile string
 var forceWrite bool
 var cacheDir string
 var LogLevel string
 var format string
+var skipTsVerifyRegistries multiValueFlag
+var registriesCertificates = make(keyValueFlag)
```

Comment:
I don't think you need to do a `make()` here, right?

Reply:
I changed the declaration to `var registriesCertificates keyValueFlag` and changed the usage in `init()`to

```go
registriesCertificates = make(keyValueFlag)
	RootCmd.PersistentFlags().VarP(&registriesCertificates, "registry-certificate", "", "Use the provided certificate for TLS communication with the given registry. Expected format is 'my.registry=/path/to/the/server/certificate'.")
```

Otherwise it would have lead to a `panic: assignment to entry in nil map` error

See [here](https://play.golang.org/p/vTXZY3pe7qC)